flickit-assessment-kit/src/test/java/org/flickit/assessment/kit/application/port/in/assessmentkit/EditKitInfoUseCaseParamTest.java

```diff
@@ -0,0 +1,91 @@
+package org.flickit.assessment.kit.application.port.in.assessmentkit;
+
+import jakarta.validation.ConstraintViolationException;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_ID_NOT_NULL;
+import static org.flickit.assessment.kit.common.ErrorMessageKey.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class EditKitInfoUseCaseParamTest {
+
+    private static final Long KIT_ID = 1L;
+    private static final String TITLE = "title";
+    private static final String MIN_TITLE = "t";
+    private static final String MAX_TITLE = RandomStringUtils.randomAlphabetic(51);
+    private static final String SUMMARY = "summary";
+    private static final String MIN_SUMMARY = "s";
+    private static final String MAX_SUMMARY = RandomStringUtils.randomAlphabetic(201);
+    private static final Boolean IS_ACTIVE = Boolean.TRUE;
+    private static final Boolean IS_PRIVATE = Boolean.FALSE;
+    private static final Double PRICE = 0D;
+    private static final String ABOUT = "about";
+    private static final String MIN_ABOUT = "a";
+    private static final String MAX_ABOUT = RandomStringUtils.randomAlphabetic(1001);
+    private static final List<Long> TAGS = List.of(2L, 3L);
+    private static final UUID CURRENT_USER_ID = UUID.randomUUID();
+
+    @Test
+    void testEditKitInfo_kitIdIsNull_ErrorMessage() {
```

Comment:
Replace `testEditKitInfo` with `testUpdateKitInfo` in all tests (both param tests and service tests)